source/indexes.txt

```diff
@@ -83,7 +83,16 @@ Create an Index
 Create an Atlas Search Index
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To create an Atlas Search index, see :ref:`compass-create-search-index`.
+Atlas Search indexes let you query data in `Atlas Search
+<https://www.mongodb.com/docs/atlas/atlas-search/>`__. To create an Atlas 
+Search index, see :ref:`compass-create-search-index`.
```

Comment:
[suggestion] I would suggest taking out some of the redundancy here. (I know this format matches the other "create a xxx index" sections on the page but would suggest changing all of them for minimalism)

```suggestion
<https://www.mongodb.com/docs/atlas/atlas-search/>`__. For more information, see :ref:`compass-create-search-index`.
```

Reply:
Updated.